backend/src/test/java/de/lowani/backend/entities/UserEntityTest.java

```diff
@@ -0,0 +1,73 @@
+package de.lowani.backend.entities;
+
+import de.lowani.backend.SpringBootTests;
+import de.lowani.backend.repo.UserRepo;
+import org.junit.jupiter.api.Test;
+import org.springframework.dao.DataIntegrityViolationException;
+
+import javax.annotation.Resource;
+import javax.transaction.Transactional;
+import java.util.HashSet;
+import java.util.Optional;
+import java.util.Set;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+class UserEntityTest extends SpringBootTests {
+    @Resource
+    private UserRepo userRepository;
+
+    @Test
+    @Transactional
+    public void testCreateUserWithoutNameShouldThrow() {
+        try {
+            UserEntity userEntity = new UserEntity();
+            userRepository.save(userEntity);
+            fail("user without name must not be persisted");
+
+        } catch (DataIntegrityViolationException ignore) {
```

Comment:
Wie genau funktioniert das?

Reply:
Das fängt den nullpointer, da man ja ein null Objekt versucht zu speichern. Sonst würde es chrashen. Is ne Spring excepion, haben wir auch so im GitHub-Beispiel gemacht